src/ducks/Signals/signalFormManager/signalCrudForm/formParts/TriggerFormHistoricalBalance.js

```diff
@@ -12,26 +12,48 @@ import {
 import { ASSETS_BY_WALLET_QUERY } from '../../../../HistoricalBalance/common/queries'
 import {
   mapAssetsHeldByAddressToProps,
-  mapToAssets,
   isPossibleEthAddress
 } from '../../../utils/utils'
+import { TriggerProjectsSelector } from './ProjectsSelector/TriggerProjectsSelector'
+import FormikSelect from '../../../../../components/formik-santiment-ui/FormikSelect'
 import styles from '../signal/TriggerForm.module.scss'
 
-const isDisabledWalletAddressField = (
-  canUseMappedErc20,
-  target,
-  allErc20Projects
-) => {
-  if (canUseMappedErc20 || !target || !target.value) {
-    return false
+const isInHeldAssets = (heldAssets, checking) => {
+  for (let i = 0; i < checking.length; i++) {
+    const checkingAsset = checking[i]
+
+    if (!heldAssets.some(({ slug }) => slug === checkingAsset.slug)) {
+      return false
+    }
```

Comment:
maybe for simplify reading use `forEach` instead `for`?

Reply:
every and some, fixed